packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -161,6 +161,9 @@ class NavigatorObserver {
   void didPop(Route<dynamic> route, Route<dynamic> previousRoute) { }
 }
 
+/// See [Navigator.popUntil].
```

Comment:
Add at least one line of explanatory material above this. I'd probably replace the whole thing with just `Signature for [Navigator.popUntil] predicate argument.`.